### PR TITLE
user edit page is added

### PR DESCRIPTION
**Pull request by Abhinaba006**
## What is the change?
A new page is added to let the user edit their details after sign up.

## Related issue?
close: #427 

## How was it tested?
locally

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Add a screenshot or demo video if appropriate.
when some when is logged in this option will be visible 
image

the user option redirect them to a new page where they can edit their details and need to provide password to confirm changes.
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "start": "node app.js",
-    "dev": "nodemon app.js"
+    "dev": "nodemon app.js -e js,ejs"
```
Comment (Pranav016) [inline package.json:11]: was this added because changes in ejs files don't trigger nodemon to restart server ?

**Review by Pranav016 — CHANGES_REQUESTED** [review]
Changes look good, change base branch to master

File: package.json (hunk 1 above)
Comment (Abhinaba006) [reply package.json:11]: Yes, it will trigger to restart the server if an ejs file is changed

**Comment by Abhinaba006** [comment]
Sir kindly add the gssoc21 and level3 label to this pr as the issue have them 😅

**Comment by Abhinaba006** [comment]
I have changed the base to master

**Comment by Pranav016** [comment]
Changes look good, I think ready to merge, one question. There was a dashboard option in navbar when user is logged in. In your screenshot its not there..
Shouldn't edit profile function be there in user dashboard..
You can add a button for edit profile in user dashboard itself.

**Comment by Abhinaba006** [comment]
> Changes look good, I think ready to merge, one question. There was a dashboard option in navbar when user is logged in. In your screenshot its not there..
> Shouldn't edit profile function be there in user dashboard..
> You can add a button for edit profile in user dashboard itself.

I have done it before the dashboard was added, and after that I pulled the master, both are working fine together. Do I need to change them ?

**Comment by Pranav016** [comment]
> > Changes look good, I think ready to merge, one question. There was a dashboard option in navbar when user is logged in. In your screenshot its not there..
> > Shouldn't edit profile function be there in user dashboard..
> > You can add a button for edit profile in user dashboard itself.
> 
> I have done it before the dashboard was added, and after that I pulled the master, both are working fine together. Do I need to change them ?

You can raise a new issue and work on it or you can work in the same one..

**Comment by Pranav016** [comment]
/level2/

**Comment by Pranav016** [comment]
/gssoc/

**Comment by Pranav016** [comment]
> Changes look good, I think ready to merge, one question. There was a dashboard option in navbar when user is logged in. In your screenshot its not there..
> Shouldn't edit profile function be there in user dashboard..
> You can add a button for edit profile in user dashboard itself.

@Abhinaba006 you can do this in a new PR

**Comment by Abhinaba006** [comment]
@Pranav016 sir, in the issue it was labeled as level3 😅 I have no problem though, should I raise a issue to merge this user  edit option to the dashboard?

**Pull request merged**